content/ngf/installation/installing-ngf/upgrading-ngf.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Upgrade NGINX Gateway Fabric to version 2.0.0
+weight: 400
+toc: true
+type: how-to
+product: NGF
+docs: DOCS-0000
+---
+
+This guide provides step-by-step instructions for upgrading NGINX Gateway Fabric from version 1.6.x to 2.0.x, highlighting key architectural changes, expected downtime, and important considerations for custom resource definitions.
+
+---
+
+### Upgrade from v1.6.x to v2.0.x
+
+To upgrade NGINX Gateway Fabric from version 1.6.2 to the latest version 2.0.0, you must uninstall the existing CRDs and perform a fresh installation with the new CRDs. This upgrade involves uninstalling the existing NGINX Gateway Fabric deployment making it a breaking change and will lead to application downtime during the process.
+
+{{<note>}} You do not need to uninstall the Gateway API CRDs during the upgrade. These standard Kubernetes resources are compatible with the new NGINX Gateway Fabric version. {{</note>}}
+
+#### Uninstalling previous NGINX Gateway Fabric v1.6.x
+
+To remove the previous version 1.6.x of NGINX Gateway Fabric, follow these steps:
+
+1. Run the following command to uninstall NGINX Gateway Fabric from the `nginx-gateway` namespace:
+
+```shell
+helm uninstall ngf -n nginx-gateway 
+```
+
+1. To remove the CRDs associated with NGINX Gateway Fabric version 1.6.x, run the following command:
+
+```shell
+kubectl delete -k "github.com/nginxinc/nginx-gateway-fabric/config/crd?ref=< ngf-version >"
+```
+
+#### Install using Helm
+
+Follow these steps to install NGINX Gateway Fabric v2.0.0 using helm:
+
+1. First, apply the new CRDs with the following command:
+
+```shell
+kubectl apply -k "github.com/nginxinc/nginx-gateway-fabric/config/crd?ref=< ngf-version >"
+```
+
+1. Next, to install the latest stable release of NGINX Gateway Fabric in the `nginx-gateway` namespace, run the following command:
+
+```shell
+helm install ngf oci://ghcr.io/nginx/charts/nginx-gateway-fabric --create-namespace -n nginx-gateway
+```
+
+For additional customization options during the helm installation process, take a look at [Installation with Helm](../../installation/installing-ngf/helm.md).
+
+#### Install using Manifests
```

Comment:
You can take advantage of the `tab` functionality in the docs to clean up the way the two methods are displayed. See the existing installation docs for examples.